dv/management/commands/import_news.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+from urllib.request import urlopen
+from datetime import datetime
+from pytz import timezone
+
+from django.core.management.base import BaseCommand
+from dv.models import News
+
+ENDPOINT = 'http://eeagrants.org/rest/articles?page={}'
+
+
+class Command(BaseCommand):
+    help = 'Import the file given as argument'
+
+    def handle(self, *args, **options):
+        page = 0
+        data = []
+        News.objects.all().delete()
```

Comment:
I know, but not worth to do, it takes a few seconds to import all. Except when endpoint fails with http 500 or timeout